packages/client/src/__testSetup__/node.ts

```diff
@@ -32,7 +37,13 @@ export const setUpOrganization: SetUpOrganization = async (doLogin = true) => {
     // Create a new user in auth
     createUserWithEmailAndPassword(auth, email, pass),
     // Set given user as admin in org structure
-    orgRef.set({ admins: [email] }),
+    orgRef.set({
+      admins: [email],
+      emailTemplates: {
+        SendBookingsLinkTemplate,
+        SendCalendarFileTemplate,
```

Comment:
I'm gonna go out on a limb and say this is the reason the tests below are failing. Just make it an object (like I've described) and it should be good to go.